.buildkite/scripts/run_tests.sh

```diff
@@ -1,15 +1 @@
-if [ -z "$ELASTICSEARCH_TREEISH" ]; then
```

Comment:
We don't need to touch es-treeish as we utilize from `gradle.properties`, similarly no need to set target branch env var as this script target branches itself.